lab_11.cpp

```diff
@@ -6,11 +6,11 @@
 
 // ------ Declarations for utility functions ------
 
-/// \brief Replace a ratio of old_samples with new_samples
-/// \param[in] old_samples Samples used in the current model.
+/// \brief Replace a ratio of current_samples with new_samples
+/// \param[in] current_samples Samples used in the current model.
 /// \param[in] new_samples New samples.
 /// \param[in] update_ratio The ratio of samples to replace on average.
-void updateSamples(cv::Mat& old_samples, const cv::Mat& new_samples, float update_ratio);
+void updateSamples(cv::Mat& current_samples, const cv::Mat& new_samples, float update_ratio);
```

Comment:
Jeg foretrekker `old_samples` fordi current en liten stund ikke er current, men old, før den blir current med new.

Reply:
Greit nok. Har egentlig ikke noen sterk preferanse.

Tenkte bare at koden ville bli enklere å lese for studentene hvis variabel navnene i implementasjonen speilet hvordan metoden ble brukt.